test/cuda/interp1d_test.cu

```diff
@@ -5,90 +5,43 @@
 #include <iostream>
```

Comment:
Same reasoning as the `nupts` test. I fixed it before realizing it was a lot of work for basically no payoff.

Reply:
Would argue that we should remove them both in that case. This also digs quite deep into the codebase (i.e., we're no longer testing the public API). Maybe that's ok though. Don't know what best practice is here.

Reply:
Yeah I think this should come up in terms of broader testing. It's normal enough to dig deep into the codebase with tests, since we don't want users messing around with this stuff, so it can't really be public. Agree that we should just axe anything that doesn't have an obvious use and think of how to have better accuracy tests to cover any/all cases.

Reply:
Ok. Maybe we can discuss at next meeting.